packages/opentelemetry-types/src/trace/span.ts

```diff
@@ -62,7 +62,7 @@ export interface Span {
    * @param [attributes] the attributes that will be added; these are
    *     associated with this event.
    */
-  addEvent(name: string, attributes?: Attributes): this;
+  addEvent(name: string, attributes?: { [key: string]: unknown }): this;
```

Comment:
My understanding is that `Attributes` is the same type as `{ [key: string]: unknown }`, or is that not correct? If so, what's the motivation for the change here

Reply:
This PR was closed right after creation because I accidentally changed another file because of autoformat. Please ignore.